partner_firstname/tests/test_copy.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+"""Test copy function for models."""
+
+from odoo.tests.common import TransactionCase
+from .base import MailInstalled
+
+
+class UserCase(TransactionCase, MailInstalled):
+    """Test ``res.users``."""
+    model = "res.users"
+    context = {"default_login": "user@example.com"}
+
+    def setUp(self):
+        super(UserCase, self).setUp()
+        self.create_original()
+
+    def create_original(self):
+        self.original = self.env["res.users"].create({
+            "firstname": u"Firstname",
+            "lastname": u"Lastname",
+            "name": u"Firstname Lastname",
+            "login": u"firstname.lastname"
+        })
+
+    def tearDown(self):
```

Comment:
No need for this method, as you do not add to the super method.

Reply:
done